x-pack/plugins/observability/public/pages/alerts/components/alerts_flyout/alerts_flyout.tsx

```diff
@@ -91,7 +92,7 @@ export function AlertsFlyout({
     {
       title: translations.alertsFlyout.lastUpdatedLabel,
       description: (
-        <span title={alertData.start.toString()}>{moment(alertData.start).format(dateFormat)}</span>
+        <span title={moment(alertData.fields[TIMESTAMP]).valueOf().toString()}>{moment(alertData.fields[TIMESTAMP]).format(dateFormat)}</span>
```

Comment:
I wonder if the `title` shouldn't be human readable? The previous implementation, it's not but I think that's being used for accessibility?

Reply:
I am not an accessibility expert, but according to https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/title, `title` is mostly used as tooltip. Should I change it to `moment(alertData.fields[TIMESTAMP]).format(dateFormat)`?

Reply:
Yes, the title is for tooltip. @simianhacker, I think it should be human-readable. 
<img width="1465" alt="Screenshot 2022-03-02 at 14 32 55" src="https://user-images.githubusercontent.com/6838659/156577092-5e339ae1-31dc-4bb3-9cb3-3f9ddb429834.png">
i

Reply:
It's correct to use the timestamp instead of alert start. It's used in the Alerts table too. @tobiasstadler Thank you for your contribution!

Reply:
I think it should be human-readable. To me, the only reason this could be useful is a case where the "Last updated" value is something like "3 min ago". This is a case where we'd like to show the exact timestamp on hover.

Reply:
Is it ok to use the date format specified in the advanced settings (as for the actual value) or should the be same fixed format?

Reply:
Yes

Reply:
Done